lib/charms/vault_k8s/v0/vault_kv.py

```diff
@@ -324,10 +339,14 @@ def _on_relation_changed(self, event: ops.RelationChangedEvent):
                 app_name=event.app.name,
                 unit_name=unit.name,
                 mount_suffix=event.relation.data[event.app]["mount_suffix"],
-                egress_subnet=event.relation.data[unit]["egress_subnet"],
+                egress_subnets=event.relation.data[unit]["egress_subnet"].split(","),
```

Comment:
Will that work if there's a space after the comma? We should likely get rid of spaces

Reply:
There should never be a space, it's the lib that's puts data into the databag only.
But I fixed it in a new function just in case.

Reply:
In the change you made to the charm relation interfaces, there's a space after the comma.

https://github.com/canonical/charm-relation-interfaces/pull/163/files

Reply:
✅